### DISCUSSION TITLE
An option to disable the summary comment on each PR

### DISCUSSION THREAD

**Discussion by armandabric** [BetaHuhn/deploy-to-vercel-action#90] [Ideas] [answered]
Hey,

We are working on a project with two deployment by PR (one for the main website, and one for the storybook). We have today too many comments on each PR. 

We want to use the https://github.com/phulsechinmay/rewritable-pr-comment action to write one deployment comment by project. For this we need an option `COMMENT=true|false` to be able to disable the comment of the action.

What did you think of that?

**Comment by shouze** [comment]
Looks like it's pretty easy to do, just need a conditionnal statement that avoid running this part:
https://github.com/BetaHuhn/deploy-to-vercel-action/blob/8c9cff3d1f2651c78aaa43b1f42b8c4885efa83b/src/index.js#L119-L142

About the input var name what about `DISABLE_AUTO_COMMMENT`? We should probably explain that in that case you can use outputs of the action to build your comment system by yourself this way.

**Reply by BetaHuhn** [reply]
> About the input var name what about DISABLE_AUTO_COMMMENT?

I'd prefer to have a `CREATE_COMMENT` option which defaults to true.

> We should probably explain that in that case you can use outputs of the action to build your comment system by yourself this way.

Yeah that might be helpful, it could be added as an [example](https://github.com/BetaHuhn/deploy-to-vercel-action#-examples).

PR welcome :)

**Comment by shouze** [comment, marked as answer]
@BetaHuhn here it is the PR https://github.com/BetaHuhn/deploy-to-vercel-action/pull/92

**Reply by BetaHuhn** [reply]
Thanks, merged and a part of [v1.5.0](https://github.com/BetaHuhn/deploy-to-vercel-action/releases/tag/v1.5.0)!
